.github/workflows/selfbuild.yml

```diff
@@ -27,6 +27,11 @@ jobs:
       - uses: actions/checkout@v3
         with:
           persist-credentials: false
+
+      - uses: taiki-e/install-action@v2
+        with:
+          tool: cargo-auditable
+
       - name: Build Thyself
```

Comment:
it's occurred to me that the "Build Thyself" step chould probably assert that `"${TEMPDIR}/${CRATE}-${VERSION}-${TARGET_ARCH}.tar.gz"` has been created and that

[[ "$(tar --list -f "${TEMPDIR}/${CRATE}-${VERSION}-${TARGET_ARCH}.tar.gz")" == "cargo-quickinstall" ]]

Reply:
I suppose it could also upload the tarball so you can fetch it and check manually?

Reply:
That's doable, but I feel perhaps it's better as a separare PR?